ScrapPackedExplorerCli/Properties/launchSettings.json

```diff
@@ -2,7 +2,8 @@
   "profiles": {
     "ScrapPackedExplorerCli": {
       "commandName": "Project",
-      "commandLineArgs": "--help"
+      //"commandLineArgs": "--help"
+      "commandLineArgs": "add -p empty.packed -s examplefile1.txt"
```

Comment:
not sure if i want to have th default launch profile to add a file to empty.packed
maybe add a new launch profile?

Reply:
Just tested stuff and forgot to remove